src/map/components/footer/footer.component.tsx

```diff
@@ -1,14 +1,20 @@
-import {Box, Typography, Link} from '@material-ui/core';
+import {Box, Typography} from '@material-ui/core';
 import React from 'react';
 import {urls} from '../../../shared/externalURL';
+import LinkExternal from '@material-ui/core/Link';
+import {Link} from 'react-router-dom';
 
 export default function Footer() {
   return (
     <Box display="flex" justifyContent="center" m={1} p={1}>
       <Typography variant="caption" display="block" gutterBottom>
         Made by{' '}
-        <Link href={urls.maibornWolff} target="/blank">
+        <LinkExternal href={urls.maibornWolff} target="/blank">
           Maibornwolff
+        </LinkExternal>
+        {' - '}
```

Comment:
```suggestion
        {' − '}
```
Just a little typography (- ⇒ −)